consensus/core/src/block.rs

```diff
@@ -613,6 +622,19 @@ impl TestBlock {
     }
 }
 
+/// A block can attach reports of misbehavior by other authorities.
+#[derive(Clone, Serialize, Deserialize, Debug)]
+pub struct MisbehaviorReport {
```

Comment:
We can use `ByzantineBehavior` instead of `Misbehavior` but it is just longer. Also I think `ByzantineBehavior` implies the behavior appears differently to different validators, which may not be the case: every validator may have observed the reported `Misbehavior`.